src/main/config/config.c

```diff
@@ -488,7 +492,7 @@ void createDefaultConfig(master_t *config)
 #endif
     config->gyro_soft_type = FILTER_PT1;
     config->gyro_soft_lpf_hz = 90;
-    config->gyro_soft_notch_hz = 0;
+    config->gyro_soft_notch_hz = 230;
```

Comment:
Well thats the discussion I now have on RCG with people for the defaults.

It doesnt seem needed, but on some noisier setups it does help. I am still
in doubt about this one tbh.

The notch filter can eliminate problem area there where lowpass stil doesnt
have enough attenuation.

Its a PR now in case I decide to do it, but I think it wont happen.